### DISCUSSION TITLE
Another Galaxy Brain achievement test

### DISCUSSION THREAD

**Discussion by augustobr1to** [Schweinepriester/github-profile-achievements#32] [Q&A] [answered]
working?

**Comment by nmcardoso** [comment]
Lets try 🙂 @augustobritodev

**Comment by JonasJs** [comment, marked as answer]
to validate the achievement, simply have 2 comments marked as an answer

**Comment by ItzJustNick** [comment]
I think you need to answer 2 disscusions and then become the accepted answer on both to get the achivement.
